CHANGELOG.md

```diff
@@ -13,6 +13,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Updated samtools/minimap2 container fixing CI issues and issues running the pipeline with Docker. See [PR 85](https://github.com/phac-nml/mikrokondo/pull/85)
 
+### `Added`
+
+- Altered name of stored `SpeciesTopHit` field in the irida-next.config, and added a field displaying the field name used addressing (Issue 81)[https://github.com/phac-nml/mikrokondo/issues/81]
```

Comment:
Can you also reference this PR (PR 90)

Reply:
fixed in: https://github.com/phac-nml/mikrokondo/pull/90/commits/d60495d41e4628d0233659b88f0baa0d0d50a549